apps/retrospective/services/task.go

```diff
@@ -76,6 +82,9 @@ func (service TaskService) GetMembers(id string, retroID string, sprintID string
 		Where("task_id = ?", id).
 		Joins("JOIN sprint_members AS sm ON sprint_member_tasks.sprint_member_id = sm.id").
 		Joins("JOIN users ON sm.member_id = users.id").
+		Joins("JOIN sprints ON sm.sprint_id = sprints.id").
+		Where("(sprints.status <> ? OR sprints.id = ?)", retroModels.DraftSprint, sprintID).
```

Comment:
let's create a scope for non-discarded sprint
fix or add todo to remove duplicate queries

Reply:
We cannot create a scope here as we need to pass in the sprint id, and GORM does not allow that.

Have done the refactoring of queries.